internal/admin/dashboard/templates/page-workflows.html

```
@@ -127,7 +127,7 @@ <h3 x-text="workflowSubmitMode() === 'save' ? 'Edit Workflow' : 'Create Workflow
                         <span>Guardrails</span>
                     </label>
                     <label class="workflow-feature-toggle" x-show="workflowFailoverVisible()">
-                        <input type="checkbox" x-model="workflowForm.features.fallback">
+                        <input type="checkbox" x-model="workflowForm.features.failover">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)